src/view/LoginView.java

```diff
@@ -14,7 +14,7 @@
 
 public class LoginView extends JPanel implements ActionListener, PropertyChangeListener {
 
-  public final String viewName = "log in";
+  public static final String viewName = "log in";
```

Comment:
This change is a big strange, what's the right way to do this? (I didn't want to pass an instance of loginview to welcomeview so that it could know the viewName of what it was switching to)